c2_utils/src/mfx_va_frame_pool_allocator.cpp

```diff
@@ -155,6 +156,13 @@ mfxStatus MfxVaFramePoolAllocator::AllocFrames(mfxFrameAllocRequest *request,
             }
             MFX_DEBUG_TRACE_I32(response->NumFrameActual);
 
+            if (MFX_ERR_NONE != mfx_res) {
+                MFX_DEBUG_TRACE_MSG("Fatal error occurred while allocating memory");
```

Comment:
Looks good to me. Did you re-scan it after this change? Is this coverity issue gone? 

Thanks,
Tianmi

Reply:
> Looks good to me. Did you re-scan it after this change? Is this coverity issue gone?
> 
> Thanks, Tianmi

Not yet, it's a bit troublesome to build the sanning environment by myself, I haven't done yet. I used to ask QA to help me execute it.